app/src/main/java/com/gmail/mxwild/newhabit/movieslist/MoviesListViewModel.kt

```diff
@@ -0,0 +1,44 @@
+package com.gmail.mxwild.newhabit.movieslist
+
+import android.content.Context
+import android.util.Log
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import com.gmail.mxwild.newhabit.model.data.Movie
+import com.gmail.mxwild.newhabit.model.data.loadMovies
+import com.gmail.mxwild.newhabit.utils.State
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.launch
+
+class MoviesListViewModel() : ViewModel() {
+
+    private val _mutableMovieList = MutableLiveData<List<Movie>>(emptyList())
+    val moviesList: LiveData<List<Movie>> get() = _mutableMovieList
```

Comment:
Why to have two live data and not just make list part of the State.Success?

Reply:
How i can do it?

Reply:
Replied in different comment